package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ava": {
+    "babel": {
+      "presets": []
+    },
+    "concurrency": 5,
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "chalk": "^2.4.1",
+    "debug": "^4.1.0",
+    "globby": "^8.0.1",
+    "optionator": "^0.8.2",
+    "request": "^2.88.0",
+    "uslug": "^1.0.4"
+  },
+  "devDependencies": {
+    "@types/debug": "0.0.31",
+    "@types/globby": "^8.0.0",
+    "@types/node": "^10.12.10",
+    "@types/request": "^2.48.1",
+    "@types/sinon": "^5.0.7",
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.9.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-markdown": "^1.0.0-rc.0",
+    "eslint-plugin-typescript": "^0.14.0",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^13.1.0",
+    "rimraf": "^2.6.2",
+    "sinon": "^7.1.1",
+    "ts-node": "^7.0.1",
+    "typescript": "^3.1.6",
+    "typescript-eslint-parser": "^21.0.1"
+  },
+  "description": "Validate markdown links",
+  "homepage": "https://github.com/webhintio/markdown-link-validator#readme",
```

Comment:
Nit: Maybe we should add a `files` property for the release to make sure we only publish the things we want. We are doing this in other repos. E.g.: 

```json
"files": [
    "dist/src",
    "images"
  ],
```